IANA registries for parameter names and transform names (#95)
ietf-wg-masque/draft-ietf-masque-quic-proxy: draft-ietf-masque-quic-proxy.md

~~~
{: #iana-header-type-table title="Registered HTTP Header"}

> ## QUIC Proxy Parameter Names

Reviewer: ```suggestion
## Proxy QUIC Forwarding Parameter Names
```

> This document establishes a new registry for QUIC proxy parameter names

Reviewer: These are parameters specifically to `proxy-quic-forwarding`, right? Perhaps we should call this all "Proxy QUIC Forwarding Parameter Names", etc?

Reviewer: ```suggestion
This document establishes a new registry, "Proxy QUIC Forwarding Parameter Names",
for parameter names to use with the `Proxy-QUIC-Forwarding` header field,
```

Registrations in this registry are assigned using the
Specification Required policy (Section 4.6 of [IANA-POLICY]).

~~~
    +-----------------------+-------------------------------------+---------------+--------------------------------+
    | Parameter Name        | Description                         | Reference     | Notes                          |
    +-----------------------+-------------------------------------+---------------+--------------------------------+
    | accept-transform      | contains supported transforms       | This document | Section {{client-behavior}}    |
    +-----------------------+-------------------------------------+---------------+--------------------------------+
    | transform             | indicates selected transforms       | This document | Section {{proxy-behavior}}     |
    +-----------------------+-------------------------------------+---------------+--------------------------------+
    | scramble-key          | contains key for scramble transform | This document | Section {{scramble-transform}} |
    +-----------------------+-------------------------------------+---------------+--------------------------------+
~~~

> {: #iana-parameter-names-table title="Initial QUIC Proxy Parameter Names"}

Reviewer: ```suggestion
{: #iana-parameter-names-table title="Initial Proxy QUIC Forwarding Parameter Names"}
```

## Packet Transform Names

This document establishes a new registry for packet transform names

> in <[](https://www.iana.org/assignments/masque/masque.xhtml)> 

Author: ```suggestion
in <[](https://www.iana.org/assignments/masque/masque.xhtml)>
```

and defines two initial transforms: "null" and "scramble".
Registrations in this registry are assigned using the
Specification Required policy (Section 4.6 of [IANA-POLICY]).

| Transform Name | Description       | Specification | Notes                          |
|:---------------|:------------------|:--------------|--------------------------------|
| null           | no transformation | This Document | Section {{null-transform}}     |

> | scramble       | simples scramble  | This Document | Section {{scramble-transform}} |

Reviewer: ```suggestion
| scramble       | AES-128 size-maintaining scramble of bytes  | This Document | Section {{scramble-transform}} |
```
